src/handlers/RF24StreamLogHandler.h

```diff
@@ -0,0 +1,51 @@
+/**
+ * @file RF24StreamLogHandler.h
+ *
+ * Created on: 2 Oct 2020
+ *     Author: Witold Markowski (wmarkow)
+ *
+ * Copyright (C)
+ *    2020        Witold Markowski (wmarkow)
+ *
+ * This General Public License does not permit incorporating your program into
+ * proprietary programs.  If your program is a subroutine library, you may
+ * consider it more useful to permit linking proprietary applications with the
+ * library.  If this is what you want to do, use the GNU Lesser General
+ * Public License instead of this License.
+ */
+
+#ifndef SRC_HANDLERS_RF24STREAMLOGHANDLER_H_
+#define SRC_HANDLERS_RF24STREAMLOGHANDLER_H_
+
+#include <RF24LogHandler.h>
+#include <Stream.h>

```

Comment:
its better to use _Print.h_ because `Stream` inherits from `Print`. Thus, there would be no consequence to the rest of the code here, and you may broaden the supported ostreams datatypes.

Reply:
Ok, Good.

Reply:
BTW the LCD matrix API on Arduino docs supports a `print()`, but no `println()` I've never used one though, so I can't back that up with any real experience.

### IMPORTANT
When I inherited from `Print` class for my PR, I had to override 2 virtual `write()` functions, and I just forwarded args back to `Print::write()`. [This is what worked for me](https://github.com/2bndy5/RF24Log/blob/daa9d5bd846215923b9269493d6d10da97ffbbff/src/PrintOut.h#L116-L130). It was a bit confusing at first...

Reply:
Done.